src/pages/Settings/Settings.test.js

```diff
@@ -0,0 +1,7 @@
+// import React from 'react'
+// import Settings from './Settings'
+// import { render } from '@testing-library/react'
+
+describe('Settings', () => {
+  it("isn't tested", () => {})
```

Comment:
what's the plan for testing this? Just leave it til we know what the real settings will be?

Reply:
I'll put some testing in, just hadn't gotten to it. I was being bad.